doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,547 @@
+all the calibration files you need, in addition to the science ones.
+In this example, the raw data are stored in the folder
+``/PypeIt-development-suite/RAW_DATA/keck_lris_red_mark4/multi_600_10000_slitmask``.
+
+The files within this folder are:
+
+.. code-block:: bash
+
+    $ ls -1
+    r230417_01033.fits
+    r230417_01064.fits
+    r230417_01072_with_maskinfo.fits
+
+This folder can include data from different datasets (e.g., more than one slitmask
+or observations in various gratings). The script :ref:`pypeit_setup`
+(see next step) will help to parse the desired dataset.
+
+.. note::
+
+        **The following is only for multi-slit datasets. For long-slit observations,
+        the user can skip this step.**
+
+        Pypeit is able to incorporate slitmask information in the reduction routine,
+        (similarly to how it is done for Keck/DEIMOS observations).
+        However, unlike DEIMOS, LRIS slitmask information is not included in the raw data,
+        and the user needs to add it to the raw data before running PypeIt. The software
+        `TILSOTUA <https://github.com/jsulli27/tilsotua>`__ can help with that, but the
+        user needs first to obtain the mask design files. The files needed are two
+        output files produced by `autoslit <https://www2.keck.hawaii.edu/inst/lris/autoslit_WMKO.html>`__
+        when creating the slitmask for the observations, and The ASCII object list file fed as input to
+        `autoslit <https://www2.keck.hawaii.edu/inst/lris/autoslit_WMKO.html>`__. Those files are processed by
+        TILSOTUA and converted into binary tables, containing the slitmask design information,
+        the object catalog and the mapping between the two. The user, subsequently, needs to
+        append the binary tables to the raw flat field image file. See :ref:`lris_slitmask`
+        for more details. In this example, the raw file ``r230417_01072_with_maskinfo.fits``
+        already includes the slitmask information.
+
+        If the slitmask information is not found in the raw data file, Pypeit will
+        still be able to reduce the data but will not be able to use the slitmask information.
+
+Run ``pypeit_setup``
+--------------------
+
+The first script to run with PypeIt is :ref:`pypeit_setup`, which examines the raw files
+and generates a sorted list and (when instructed) one :ref:`pypeit_file` per instrument configuration.
+
+See complete instructions provided in :ref:`setup_doc`.
+
+.. important::
+
+    Know the spectrograph you need to use! LRIS went through different upgrades
+    (FITS file format and detectors, see :ref:`lris`) and PypeIt uses different spectrograph
+    names to differentiate between them. In this example, we are using observations taken
+    with the new Mark IV detector in the red arm, so we use the spectrograph name
+    ``keck_lris_red_mark4``. PypeIt will throw an error if you use the wrong spectrograph name.
```

Comment:
How robust is this?  Maybe we should instead say, "PypeIt attempts to detect if you've used the wrong spectrograph name and will throw an error if it thinks you have; however, we don't guarantee that this always works (hence the need to specify the spectrograph to begin with)."  But maybe this is very robust for LRIS?

Reply:
Thanks for the review. This is actually very robust for LRIS, since there is a specific script that check for that. Put it works only within LRIS, so I clarified the statement saying: "PypeIt will throw an error if you use the wrong **LRIS** spectrograph name."